aws/autoscaling.go

```diff
@@ -54,12 +45,12 @@ func NewAutoScalingGroup(region string, asg *autoscaling.Group) *AutoScalingGrou
 	}
 
 	for i := 0; i < len(asg.Tags); i++ {
-		a.Tags[*asg.Tags[i].Key] = *asg.Tags[i].Value
+		a.AWSResource.Tags[*asg.Tags[i].Key] = *asg.Tags[i].Value
```

Comment:
Both structs have a Tags field, so specify the AWSResource field, which is a map and not a struct of []ec2.Tag